language-design.md

```diff
@@ -62,7 +175,50 @@ the commenters?
 
 **Response**
 
-
+Pavlus is arguing that programming languages should be designed to be easier
+for the average person to understand and learn. While we agree with this
+general argument of the article, the actual specifics Pavlus uses to bolster
+his claim are not convincing. The commenters point out that Pavlus is
+emphasizing the difficulty of learning a language for those new to the
+language, but they argue that a programming language should be designed more
+with its users in mind, in that ease of use should be the concern, not ease
+of learning.
+
+We mostly agree with the commenters. The language Pavlus seems to be after is a
+language for teaching programming, not one for actually programming. The reason
+that his "hieroglyphic" programming languages stick around is that they are
+useful. The commenters are right that there is a trade-off between a language
+being readable by the general public and how efficient and powerful a language
+is. We resent Pavlus for implying that such a trade-off need not be there, and
+that contributors to programming language design don't care enough to make their
+languages usable. Does he think that nobody has thought of this before?
+
+However, we do not totally disagree with Pavlus. We actually agree with his
+general arguments, just not with how he defends them. We believe that it would
+be possible for languages to be more accessible to the general public. For
+example, Python has had great success with being a language designed with a
+guiding principle of "Readability counts," as well as  "Beautiful is better 
+than ugly" ([PEP 20 -- The Zen of Python][1]). It is not too hard to imagine a
+language that goes a bit farther than Python while still being very powerful
+and usable.
+
+We also note that both Pavlus and the commenters are focusing too heavily on
+one set of users. Pavlus seems only to care about having as many people as
+possible understanding a program. This to us seems silly, it is a career for a
+reason: you wouldn't expect the whole public to easily become good lawyers.
+Maybe at some point in the future programming will become more commonplace, but
+we are not there yet. Likewise, the commenters are fully rejecting the idea
```

Comment:
Well with a broad definition of programming language, maybe more people are programming than we might think. Tons of people type formulas into Excel spreadsheets and use WolframAlpha to do computations. I would even argue that microwaving my food involves programming --- and it's a bad language because I can't seem to figure out how the reheat button works.